catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/framework/FrameworkProducer.java

```diff
@@ -148,8 +148,8 @@ public void process(Exchange exchange) throws FrameworkProducerException {
       exchange.getIn().setBody(new ArrayList<Metacard>());
       LOGGER.debug("Received a non-String as the operation type");
```

Comment:
❓Should we throw an exception here as well instead of swallowing the `CastClassException`?